nats-server/Dockerfile

```diff
@@ -21,7 +21,7 @@ ARG BUILD_NUMBER=-1
 LABEL io.astronomer.docker=true
 LABEL io.astronomer.docker.build.number=$BUILD_NUMBER
 
-ENV NATS_SERVER 2.10.10
+ENV NATS_SERVER 2.10.17
```

Comment:
can you downgrade this to 2.10.14

Reply:
> can you downgrade this to 2.10.14

Sure, but the latest version is 2.10.17 right, any specific reason

Reply:
@pgvishnuram i agree with @himabindu07 that i'd like to know why we cannot upgrade to the latest.